test/integration/crypto-verify.test.ts

```diff
@@ -41,6 +36,12 @@ export function convertStringToBase64(str: string): string {
 
 describe('crypto-verify', () => {
   beforeEach(async () => {
+    const backend: IBackend = {
```

Comment:
We should recreate the backend for every test, otherwise tests will share state, which is bad.